devel/ddev/Portfile

```
@@ -0,0 +1,335 @@
+                    golang.org/x/net v0.22.0 \
+                        h1:9sGLhx7iRIHEiX0oAJ3MRZMUCElJgy7Br1nO+AMN3Tc= \
+                    golang.org/x/oauth2 v0.15.0 \
+                        h1:s8pnnxNVzjWyrvYdFUQq5llS1PX2zhPXmccZv99h7uQ= \
+                    golang.org/x/sync v0.6.0 \
+                        h1:5BMeUDZ7vkXGfEr1x9U/LP0PkLCS9tbxHdi/U= \
+                    golang.org/x/sys v0.18.0 \
+                        h1:DBdB3niSjOA/O0blCZBqDefyWNYveAYMNF1Wum0DYQ4= \
+                    golang.org/x/telemetry v0.0.0-20240228155512-f48c80bd79b2 \
+                        h1:IRJeR9r1pYWsHKTRe/IInb7lYvbBVIqOgsX/u0mbOWY= \
+                    golang.org/x/term v0.16.0 \
+                        h1:m+B6fahuftsE9qjo0VWp2FW0mB3MTJvR0BaMQrq0pmE= \
+                    golang.org/x/text v0.14.0 \
+                        h1:ScX5w1eTa3QqT8oi6+ziP7dTV1S2+ALU0bI+0zXKWiQ= \
+                    golang.org/x/time v0.5.0 \
+                        h1:o7cqy6amK/52YcAKIPlM3a+Fpj35zvRj2TP+e1xFSfk= \
+                    golang.org/x/tools v0.19.0 \
+                        h1:tfGCXNR1OsFG+sVdLAitlpjAvD/I6dHDKnYrpEZUHkw= \
+                    golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 \
+                        h1:go1bK/D/BFZV2I8cIQd1NKEZ+0owSTG1fDTci4IqFcE= \
+                    google.golang.org/appengine v1.6.8 \
+                        h1:IhEN5q69dyKagZPYMSdIjS2HqprW324FRQZJcGqPAsM= \
+                    google.golang.org/genproto/googleapis/api v0.0.0-20231002182017-d307bd883b97 \
+                        h1:W18sezcAYs+3tDZX4F80yctqa12jcP1PUS2gQu1zTPU= \
+                    google.golang.org/genproto/googleapis/rpc v0.0.0-20231002182017-d307bd883b97 \
+                        h1:6GQBEOdGkX6MMTLT9V+TjtIRZCw9VPD5Z+yHY9wMgS0= \
+                    google.golang.org/grpc v1.60.1 \
+                        h1:26+wFr+cNqSGFcOXcabYC0lUVJVRa2Sb2ortSK7VrEU= \
+                    google.golang.org/protobuf v1.33.0 \
+                        h1:uNO2rsAINq/JlFpSdYEKIZ0uKD/R9cpdv0T+yoGwGmI= \
+                    gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c \
+                        h1:Hei/4ADfdWqJk1ZMxUNpqntNwaWcugrBjAiHlqqRiVk= \
+                    gopkg.in/yaml.v2 v2.3.0 \
+                        h1:clyUAQHOM3G0M3f5vQj7LuJrETvjVot3Z5el9nffUtU= \
+                    gopkg.in/yaml.v3 v3.0.1 \
+                        h1:fxVm/GzAzEWqLHuvctI91KS9hhNmmWOoWu0XTYJS7CA= \
+                    gotest.tools/v3 v3.4.0 \
+                        h1:ZazjZUfuVeZGLAmlKKuyv3IKP5orXcwtOwDQH6YVr6o= \
+                    muzzammil.xyz/jsonc v1.0.0 \
+                        h1:B6kaT3wHueZ87mPz3q1nFuM1BlL32IG0wcq0/uOsQ18=
+
+# Dependencies
+depends_build       port:go \
+                    port:gmake
+
+depends_run         port:mkcert
+
+# Build configuration
+build.env-append    CGO_ENABLED=0 \
+                    GOFLAGS="-mod=vendor -buildvcs=false"
+
+build.cmd           make
```

`make` is usually present out of the box with MacPorts, but since you're including `gmake` a build dep, I'm assuming you want to use `gmake` here:

```suggestion
build.cmd           ${prefix}/bin/gmake
```